scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -141,6 +154,139 @@ def release_paths(self) -> List[Path]:
         return release_paths
 
 
+class ModelConfig(Config):
+    """
+
+    Example:
+    
+        name: bert-base-uncased
+        path: # should be string or should contain package details
+            package: # if package not specificed , path would be be used like os path object.
+                url: https://huggingface.co/bert-base-uncased
+                commit_hash: 5546055f03398095e385d7dc625e636cc8910bf2
+        publish:
+            type: mlflow_model # could be one of (custom_model, mlflow_model, triton_model)
+            flavor: hftransformers # flavors should be specificed only for mlflow_model and model would be published in this flavor
+            tags: # Key value pairs which will be published as tags in the registry 
+                isv : huggingface
+                task: fill-mask
+    """
+
+    @property
+    def name(self) ->str: 
+        return self._yaml.get("name")

```

Comment:
Please use single quotes around key values. Also applies to the rest of the changes in this PR.